arctic/arctic.py

```diff
@@ -101,10 +106,13 @@ def __init__(self, mongo_host, app_name=APPLICATION_NAME, allow_secondary=False,
         self._server_selection_timeout = serverSelectionTimeoutMS
         self._lock = threading.RLock()
         self._pid = os.getpid()
+        self._pymongo_kwargs = kwargs
```

Comment:
Why is this called `_pymongo_kwargs` again?

Reply:
The kwargs are getting stored for use in _conn, where they are passed to pymongo.MongoClient, and these kwargs are not used anywhere else so I was hoping it would be ok to be more explicit with the name.
Can change to self._kwargs if you prefer?